web/src/ferrostar-core.ts

```diff
@@ -137,6 +139,14 @@ class FerrostarCore extends LitElement {
     this.currentLocationMapMarker!.setLatLng(this.locationProvider.lastLocation.coordinates);
   }
 
+  private resetMap() {
+    this.map!.eachLayer((layer) => {
+      if (layer instanceof L.Marker || layer instanceof L.Polyline) {
+        this.map!.removeLayer(layer);
+      }
+    });
+  }
```

Comment:
This isn't great, but I don't know the best way to do it in Leaflet so I'll merge anyways ;) We can revisit after switching to MapLibre GL JS.